src/erlang_ls.erl

```diff
@@ -12,6 +12,7 @@ main(Args) ->
   ok = parse_args(Args),
   ok = init_node_name(is_debug()),
   ok = lager_config(),
+  els_db:install(),
```

Comment:
The DB is currently recreated on startup. Obviously this needs to be fixed.

Reply:
What happens in debug mode when the node's name is always different? Have you tried this branch out with different `erlang_ls` instances working at the same time?

Reply:
Things will probably explode. That's one of the reasons why this branch is not ready for merge :)